utils/istio/istio.go

```diff
@@ -64,19 +64,42 @@ func GetVirtualServiceNamespaceName(vsv string) (string, string) {
 	return namespace, name
 }
 
+func MultipleVirtualServiceConfigured(rollout *v1alpha1.Rollout) bool {
+	if rollout.Spec.Strategy.Canary.TrafficRouting.Istio.VirtualServices != nil {
+		return true
+	}
+	return false
+}
+
 // GetRolloutVirtualServiceKeys gets the referenced VirtualService and its namespace from a Rollout
 func GetRolloutVirtualServiceKeys(ro *v1alpha1.Rollout) []string {
+	var virtualServices []v1alpha1.IstioVirtualService
+	var virtualServiceKeys []string
```

Comment:
Are we getting keys or just `namespace+name`? The variable name is kind of confusing.

Reply:
Basically, we get the list of keys. each key will be in the format of <namespace>/<virtualService Name>

Reply:
so how about vsvsNamespaceName? Key seems have different meanings in k8s.

Reply:
not resolved yet

Reply:
The only way to identify the rollout is with namespace and name. So, having the key as the variable name would make more sense. 
I didn't change this in my latest update. Let me know your thoughts.